lib/tapjoy/autoscaling_bootstrap.rb

```diff
@@ -154,13 +154,22 @@ def confirm_config(keypair:, zones:, security_groups:, instance_type:,
       end
 
       # configure environment
-      def configure_environment(filename, env=nil, config_dir)
-        defaults_hash = self.load_yaml("#{config_dir}/config/common/defaults.yaml")
-        facet_hash    = self.load_yaml("#{config_dir}/config/clusters/#{filename}")
+
+      def configure_environment(filename, env=nil, config_dir=nil)
+        if filename.include?(File::SEPARATOR)
+          facet_file    = filename
+          config_dir   = "#{File.expand_path('../../..', facet_file)}"
+        else
+          facet_file    = File.join(config_dir, 'config', 'clusters', filename)
```

Comment:
if config dir is nil, it'll cause some issues (plus you would lose the leading slash on 'config' from your prior example, if it didn't already have one)

you could likely do something like

```ruby
File.join(config_dir || "/",'config','clusters',filename)
```

Which would fallback to using a vanilla leading slash only if config_dir is nil